doc/manual_experimental.md

```diff
@@ -89,16 +89,13 @@ No Unicode normalization step is performed.
 Overloadable enum value names
```

Comment:
Shouldn't this section be moved to the manual? Or at least removed from the `experimental` page, since it's no longer experimental?

Reply:
Wasn't sure of the best way to do that, just copy and pasting it doesn't seem adequate to me, can do in followup

Reply:
Done in #20342